### ISSUE TITLE
how to name the pictures?

### ISSUE THREAD

**Issue by Kevin35Day** [agjayant/caffe-Person-ReID#61]
Hi,
I am studying your project recnetly,
I want to train my own data on your net ,
however, I'm confused about the rule to name the pictures.
Because of this ,I am not sure the form of trainlist.txt.   
Can you share your list with us?
thanks!

**Comment by Kevin35Day** [comment]
I copyed your pictures in ReID/sampleResults as trainfile,and run the ran market_trainlist_script.sh.
I think I figure it out now.

**Comment by Kevin35Day** [comment]
I can run this net now.
I0410 15:06:22.376938  4774 solver.cpp:243] Iteration 0, loss = 1
I0410 15:06:22.376998  4774 solver.cpp:590] Iteration 0, lr = 0.001
I0410 15:06:23.158200  4774 solver.cpp:243] Iteration 5, loss = 1
I0410 15:06:23.158249  4774 solver.cpp:590] Iteration 5, lr = 0.001
I0410 15:06:23.910199  4774 solver.cpp:243] Iteration 10, loss = 1
I0410 15:06:23.910249  4774 solver.cpp:590] Iteration 10, lr = 0.001
I0410 15:06:24.658761  4774 solver.cpp:243] Iteration 15, loss = 1
I0410 15:06:24.658802  4774 solver.cpp:590] Iteration 15, lr = 0.001
I0410 15:06:25.411391  4774 solver.cpp:243] Iteration 20, loss = 1
I0410 15:06:25.411442  4774 solver.cpp:590] Iteration 20, lr = 0.001
I0410 15:06:26.164330  4774 solver.cpp:243] Iteration 25, loss = 1
I0410 15:06:26.164378  4774 solver.cpp:590] Iteration 25, lr = 0.001
I0410 15:06:26.914719  4774 solver.cpp:243] Iteration 30, loss = 1
I0410 15:06:26.914769  4774 solver.cpp:590] Iteration 30, lr = 0.001
I0410 15:06:27.669313  4774 solver.cpp:243] Iteration 35, loss = 0.999999
I0410 15:06:27.669371  4774 solver.cpp:590] Iteration 35, lr = 0.001
I0410 15:06:28.422389  4774 solver.cpp:243] Iteration 40, loss = 0.999999
I0410 15:06:28.422438  4774 solver.cpp:590] Iteration 40, lr = 0.001
I0410 15:06:29.174583  4774 solver.cpp:243] Iteration 45, loss = 1
I think I name the picture incorrectly...
Maybe because I didn't make the pictures into pairs...
Plese help me,thanks a lot

**Comment by agjayant** [comment]
Are you trying to train the models in _rank_alexnet/_ ?
If yes, then the format is simply:
image_path and it's label separated by a space.

```
/path/to/identity1/image1 label_identity1
/path/to/identity1/image2 label_identity1
...
```

You can refer to: https://dl.dropboxusercontent.com/u/334666754/unsup_patches/trainlist.txt

**Comment by Kevin35Day** [comment]
@agjayant Thank you,that helps!

**Comment by Kevin35Day** [comment]
Sorry to bother you again,I still can not train my own data correctly.
I am wondering that maybe I use your code incorrectly.Because I found serval solver.prototxt files in your REID folder,seems you used serval nets?
Could you please spare some time to introduce your ReID Project's pipeline on your github?
Like,your data's form,your trainlist,run which script and so on.
Thank you!
